src/components/SearchResults.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+
+import Bookings from "./Bookings";
+import CustomerProfile from "./CustomerProfile";
+import TableRow from "./TableRow";
+function SearchResults({ results }) {
+  const [customerId, setCustomerId] = useState(null);
+  const sortedResults = [...results];
+  const [sortedField, setSortedField] = useState(null);
+  if (sortedField !== null) {
+    sortedResults.sort((a, b) => {
+      if (a[sortedField] < b[sortedField]) {
+        return -1;
+      }
+      if (a[sortedField] > b[sortedField]) {
+        return 1;
+      }
+      return 0;
+    });
+  }
+
+  return (
+    <div>
+      <table className="table">
+        <thead className="thead-dark">
+          <tr>
+            <th scope="col">
+              <button type="button" onClick={() => setSortedField("id")}>
```

Comment:
Some of the columns are mixed up here - I think you mixed around the `title` and `firstName` in the table